src/lang/de.json

```diff
@@ -12,13 +12,34 @@
 		"CLASSES": "@:PAGE_TITLE.CLASSES",
 		"START": "@:PAGE_TITLE.START",
 		"ABOUT": "@:PAGE_TITLE.ABOUT",
-		"ITEM_SEARCH": "Suche nach Item"
+		"ITEM_SEARCH": "Suche nach Item",
+		"AUTHORIZE" : "Anmelden",
+		"UNAUTHORIZE" : "Abmelden"
 	},
 	"FOOTER": {
 		"STAT_DATE": "Statistiken Stand {{date}}",
 		"STAT_LINK": "Details",
 		"POWERED_BY": "Powered by <a href=\"https://github.com/Wikidata/Wikidata-Toolkit\">Wikidata Toolkit</a> &amp; <a href=\"https://query.wikidata.org/\">Wikidata SPARQL Query</a>"
 	},
+	"EDITS": {
+		"EDIT_LABEL": "Bezeichner ändern"	
+	},
+	"MODALS": {
+		"CLOSE": "Schließen",
+		"SAVE_CHANGES": "Änderungen speichern",
+		"EMPTY_FIELDS_ERROR": "Bitte füllen sie alle Eingabefelder aus!",
```

Comment:
It would be good if translations could avoid choosing either colloquial (du) or formal (Sie) in the address of the user. Right now, some messages are formal and others are not. There is also a typo in the above line ("sie"). I will push some updates to the translations directly.